doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -395,10 +394,10 @@ sub-directory`_ and are run via the `teuthology framework`_.
 The Ceph community has access to the `Sepia lab
 <https://wiki.sepia.ceph.com/doku.php>`_ where `integration tests`_ can be run
 on physical hardware.
-Other developers may add tags like "needs-qa" to your PR.  This allows PRs that
-need testing to be merged into a single branch and tested all at the same time.
-Since teuthology suites can take hours (even days in some cases) to run, this
-can save a lot of time.
+Other developers might add tags like "needs-qa" to your PR. This allows PRs
+that need testing to be merged into a single branch and then tested all at the
+same time. Teuthology test suites can take hours (and even days in some cases)
+to run, so batching tests in this way can save a lot of time.
```

Comment:
Other contributors might add tags like ``needs-qa`` to your PR. This allows PRs
to be merged into a single branch then tested together.
Teuthology test suites can take hours (and even days in some cases)
to complete, so batching tests reduces contention for resources and saves a lot of time.

Reply:
I'll take it. I'm rewording it to this.